api/docs/release.dox

```diff
@@ -137,7 +137,9 @@ changes:
 
 Further non-compatibility-affecting changes include:
  - Added X64 Linux support to dr_create_memory_dump(). This API has the same
-   restriction as dr_suspend_all_other_threads_ex().
+   restriction as dr_suspend_all_other_threads_ex(). For X86_64 platform, the feature is
+   supported only when fast FP save and restore is supported. And mixed mode is not
+   supported.
```

Comment:
This should go into the actual header docs for dr_create_memory_dump() itself, which is what users who want to use it will look at: they will not see this changelist.  OK to leave restrictions here too but that is not necessary: generally the changelist is just a pointer to the API docs.

Reply:
Done.